editoast/src/views/infra/attached.rs

```diff
@@ -0,0 +1,137 @@
+use std::collections::HashMap;
+
+use crate::error::Result;
+use crate::infra::Infra;
+use crate::infra_cache::InfraCache;
+use crate::schema::ObjectType;
+use crate::DbPool;
+use actix_web::dev::HttpServiceFactory;
+use actix_web::get;
+use actix_web::web::{block, Data, Json, Path};
+use chashmap::CHashMap;
+use editoast_derive::EditoastError;
+use serde_json::{json, Map, Value};
+use thiserror::Error;
+
+/// Objects types that can be attached to a track
+const ATTACHED_OBJECTS_TYPES: &[ObjectType] = &[
+    ObjectType::Signal,
+    ObjectType::SpeedSection,
+    ObjectType::Detector,
+    ObjectType::TrackSectionLink,
+    ObjectType::Switch,
+    ObjectType::BufferStop,
+    ObjectType::OperationalPoint,
+    ObjectType::Catenary,
+];
+
+/// Return `/infra/<infra_id>/attached` routes
```

Comment:
Shouldn't it be 
```suggestion
/// Return `/infra/<infra_id>/attached/<track_id>` routes
```
?

Reply:
Not really, we could have a list of endpoint starting by: `/infra/<infra_id>/attached`